docs/data.md

```diff
@@ -40,6 +40,18 @@ The following ecosystems have vulnerabilities encoded in this format:
     ([CC0 1.0](https://github.com/cloudsecurityalliance/gsd-database/blob/main/LICENSE))
 -   [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
     ([CC-BY 4.0](https://github.com/google/oss-fuzz-vulns/blob/main/LICENSE))
+    ([MIT](https://github.com/loopbackio/security/blob/main/LICENSE))
+-   [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+    ([BSD](https://rockylinux.org/licensing))
+-   [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+    ([MIT](https://github.com/AlmaLinux/osv-database/blob/master/LICENSE))
+-   [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+    ([CC0 1.0](https://github.com/haskell/security-advisories/blob/main/LICENSE.txt))
+    ([Apache 2.0](https://github.com/bitnami/vulndb/blob/main/LICENSE.md))
+-   [RConsortium Advisory Database](https://github.com/RConsortium/r-advisory-database)
+    ([Apache 2.0](https://github.com/RConsortium/r-advisory-database/blob/main/LICENSE))
+-   [Python Software Foundation Database](https://github.com/psf/advisory-database)
+    ([CC-BY 4.0](https://github.com/psf/advisory-database/blob/main/LICENSE))
 
 ## Converted data
```

Comment:
I think we've blurred reality a little bit in this section. We're not maintaining conversion pipelines for:

- AlmaLinux
- Bitnami I'm confused about completely
- crates.io
- GitHub Actions
- Go I'm unclear on the accuracy of this statement
- Haskell
- Hex
- Linux Kernel
- Maven
- npm
- NuGet
- Packagist
- Photon OS
- Pub
- Python
- R (CRAN and Bioconductor)

These all either come via GitHub Security Advisories or native maintained record sources

Reply:
So I am a little confused. I think you didn't look at the changes in context. There is a whole section between the conversion pipeline section and these changes. These are just the covered ecosystems.

Reply:
@andrewpollock See the [preview here](https://hayleycd.github.io/osv.dev/data/#converted-data) and let me know if it looks ok in the context of the whole page.

Reply:
My apologies!